src/Microsoft.ML.StandardLearners/FactorizationMachine/FactorizationMachineTrainer.cs

```diff
@@ -156,12 +156,12 @@ internal FieldAwareFactorizationMachineTrainer(IHostEnvironment env, Options opt
         /// <param name="env">The private instance of <see cref="IHostEnvironment"/>.</param>
         /// <param name="featureColumns">The name of column hosting the features. The i-th element stores feature column of the i-th field.</param>
         /// <param name="labelColumn">The name of the label column.</param>
-        /// <param name="weights">The name of the optional weights' column.</param>
+        /// <param name="weightColumn">The name of the optional weights' column.</param>
```

Comment:
> /// <param name="weightColumn">The name of the optional weights' column.</param> [](start = 7, length = 81)

/// <param name="exampleWeightColumnName">The name of the example weight column (optional).</param>